### ISSUE TITLE
Sway doesn't merge outputs with negative position

### ISSUE THREAD

**Issue by abyss7** [swaywm/sway#3268]
I want to have configs on my laptop for every monitor that I connect at home/work. So, I want my main screen to be at pos 0,0 while other screens are placed above like pos 0,-2160.

It simply doesn't work - I can't reach top screen over the top edge of my laptop with cursor.

sway version: Dec 2 2018, branch '' - actually it's a RPM package sway-1.0-0.beta.2+5.g7702943e.fc29.x86_64 from here: https://copr.fedorainfracloud.org/coprs/rzl/sway/

**Comment by emersion** [comment]
Note that Xwayland has issues with outputs at negative positions anyway. I'd suggest adding 2160 to all your x coords instead.

**Comment by RedSoxFan** [comment]
Does the following work?
```
output eDP-1 position 0 1080 scale 1 mode 1920x1080
output "Dell Inc. DELL U2718Q FN84K774130L" position 0 0 scale 2 mode 3840x2160
```

**Comment by emersion** [comment]
Ping

**Comment by abyss7** [comment]
Sorry, for losing touch.

I tried and it works - not sure if it's intended and expected behavior.

**Comment by RedSoxFan** [comment]
Yes, it is intended and expected behavior. See the wiki section: https://github.com/swaywm/sway/wiki#hidpi

This should probably be added to sway-output(5) though. I'll leave this open as a reminder to do it later

**Comment by barakt11** [comment]
Hey

> output "Dell Inc. DELL U2718Q FN84K774130L" position 0 0 scale 2 mode 3840x2160
Is not working on sway, it will work only for "eDP-1" (which changes dynamically) and not the actual device name, causes reconfig anytime it is changed.
i3 supporting name of device name like 

> workspace 2, move workspace to output 'Dell Inc. DELL U2412M YPPY089T1BWB'
Currently im running sway and still using i3-msg to support those.
